src/routes/InventoryTable.js

```diff
@@ -172,29 +181,29 @@ const Inventory = ({
                                     isFullView
                                     store={store}
                                     ref={inventory}
-                                    hasCheckbox
                                     showTags
                                     onRefresh={onRefresh}
-                                    actions={ [
-                                        {
-                                            title: 'Delete',
-                                            onClick: (_event, _index, { id: systemId, display_name: displayName }) => {
-                                                activateSystem(() => ({
-                                                    id: systemId,
-                                                    displayName
-                                                }));
-                                                handleModalToggle(() => true);
-                                            }
-                                        }, {
-                                            title: 'Edit',
-                                            onClick: (_event, _index, data) => {
-                                                activateSystem(() => data);
-                                                onEditOpen(() => true);
+                                    hasCheckbox={false}
```

Comment:
It would probably be more straight forward to do `hasCheckbox={canPerformActions}` which would indicate that if user has access to actions show the checkbox.